yeonjae/programmers-2/주차요금계산.js

```diff
@@ -0,0 +1,60 @@
+const solution = (fees, records) => {
+  const [limitTime, feePerLimit, minute, feePerMinute] = fees;
+  const answer = [];
+  const carsNumber = [];
+  const inCars = new Map();
+  const outCars = new Map();
+
+  for (const record of records) {
+    const [time, number, state] = record.split(" ");
+
+    carsNumber.push(number);
+
+    if (state === "IN") {
+      inCars.set(number, time);
+    } else {
+      const [inHour, inMinute] = inCars.get(number).split(":");
+      const [outHour, outMinute] = time.split(":");
+      const inTotal = parseInt(inHour) * 60 + parseInt(inMinute);
+      const outTotal = parseInt(outHour) * 60 + parseInt(outMinute);
```

Comment:
```suggestion
  const getConvertedMinutes = (time) => {
    const [hour, minute] = time.split(":").map(Number);

    return hour * 60 + minute;
  };
  
  // 실제 사용
  const inTotal = getConvertedMinutes(inCars.get(number));
  const outTotal = getConvertedMinutes(time);
  const savedTime = outCars.get(number) || 0;
```
- 반복적인 코드를 함수로 분리하면 깔끔할 것 같습니다!